readthedocs/projects/models.py

```diff
@@ -632,6 +632,11 @@ def save(self, *args, **kwargs):
         if self.remote_repository and not self.remote_repository.private:
             self.external_builds_privacy_level = PUBLIC
 
+        # If the project is linked to a remote repository,
+        # make sure to use the clone URL from the repository.
+        if self.remote_repository:
+            self.repo = self.remote_repository.clone_url
```

Comment:
If we want to copy this URL, we will need to add a post signal that each time the RemoteRepository is save to update the `Project.repo` as well. Otherwise, it will be out of sync when that happens.